python/lsst/sims/catUtils/mixins/AstrometryMixin.py

```diff
@@ -147,12 +154,29 @@ def get_phoSimCoordinates(self):
     def get_observedCoordinates(self):
         return self.observedStellarCoordinates()
 
+    @compound('raICRS', 'decICRS')
```

Comment:
I think the docstring for this mixin should be updated as well since it seems that originally it only contained one getter but obviously contains multiple now.